bigc/resources/orders.py

```diff
@@ -114,6 +114,10 @@ def update_shipment(self, order_id: int, shipment_id: int, **kwargs) -> dict:
         """Updates an order shipment for the specified order"""
         return self._api.v2.put(f'/orders/{order_id}/shipments/{shipment_id}', json=kwargs)
 
+    def delete_shipment(self, order_id: int, shipment_id: int, **kwargs) -> None:
+        """Deletes specific shipment by its ID"""
+        return self._api.v2.delete(f'/orders/{order_id}/shipments/{shipment_id}', json=kwargs)
```

Comment:
The code here is doing too much, see https://github.com/MedShift/bigc/blob/3a0db1a82b1415e9dac6fbf6ae7dec99f4603d03/bigc/resources/categories.py#L33

Reply:
[fix the doing too much](https://github.com/MedShift/bigc/pull/42/commits/edf9a6e1c98f49a473515e51029b6510f07c7733) REFER HERE